File: 2529-maximum-count-of-positive-integer-and-negative-integer/2529-maximum-count-of-positive-integer-and-negative-integer.py
class Solution:
    def maximumCount(self, nums: List[int]) -> int:
        p = 0
        n = 0

        for num in nums:
            if num > 0:
                p += 1
            if num < 0:
                n += 1
        
        return max(p, n)

File: 2529-maximum-count-of-positive-integer-and-negative-integer/2529-maximum-count-of-positive-integer-and-negative-integer.go
func maximumCount(nums []int) int {
    var n, p int
    n, p = 0, 0

    for _, num := range nums {
        if num < 0 {
            n++
        } else if num > 0 {
            p++
        }
    }

    return max(n, p)
}